src/signals/incident/components/form/DateTimeInput/index.js

```diff
@@ -86,21 +86,23 @@ const DateTimeInput = ({ touched, hasError, meta, parent, getError, validatorsOr
 
         <TimeWrapper>
           <div>
-            <StyledLabel htmlFor="hours">Tijd</StyledLabel>
+            <StyledLabel>Tijd</StyledLabel>
```

Comment:
Wrapping `uur` and `min` in a `fieldset` with `legend` (Tijd) would associate the `Tijd` label better with the input components. What do you think?